ctapipe/io/nectarcameventsource.py

```diff
@@ -242,3 +247,83 @@ def fill_r0_container_from_zfile(self, event):
             event
         )
 
+class MultiFiles:
+
+    """
+    This class open all the files in file_list and read the events following
+    the event_id order
```

Comment:
Could use a better description and class name - does this merge multiple camera files? Or chain together multiple observations? I guess the former?  Does it provide an EventSource interface? Does it keep all files in memory?

Reply:
I would not do this here, the nectarcam got also removed, see #993

Reply:
So, though this class was moved here, because nectarcam dependent on lstcam internals, (and also sst1m internals, btw), we should ignore this and just merge both PRs.